bugwarrior/services/github.py

```diff
@@ -49,6 +49,17 @@ def get_issues(self, username, repo):
             username=username, repo=repo)
         return self._getter(url)
 
+    def get_issue_from_path(self, issue_path):
+        """Get issue from issue_path."""
+        # TODO: test new scheme of issue_path
+        regex = r'(^[^/]+)(/)([^/]+)(/issues/[0-9]+)'
+        if not re.match(regex, issue_path):
```

Comment:
Can you do this validation step in the ``__init__`` method if `include_these_issues` is set?  That should catch things earlier.

Reply:
Good idea. Unfortunately, I've tried that by reading these issues from the config with `self.config.get('include_these_issues')` in `__init__` but I get an error:
```
AttributeError: 'GithubClient' object has no attribute 'config'
```

Reply:
Ah - try instead in the `__init__` method of the `GithubService` class instead of the `GithubClient` class.  See where all those `self.client....` calls are concentrated?